client/jetpack-cloud/sections/agency-dashboard/sites-overview/style.scss

```diff
@@ -35,13 +35,51 @@
 		border-bottom: 1px solid var(--color-primary-5);
 	}
 }
+.sites-overview__add-site-issue-license-buttons {
+	// We need these negative margin values because we want to make the container full-width,
+	// but our element is inside a limited-width parent.
```

Comment:
Very nice leaving a comment like this, to explain the use of negative margins. (They always confuse me!)

Reply:
Sadly, that one wasn't me, it was an existing comment I added to the new block since it behaved the same way, but thanks :)